HaikuPorter/Source.py

```diff
@@ -366,12 +366,14 @@ def patch(self, port):
 		if patched:
 			port.setFlag('patchset', self.index)
 
-		return patched
-
-	def reset(self):
+	def reset(self, resetToOrigin=True):
```

Comment:
Should I maybe change the parameter to a string defaulting to 'ORIGIN' and call it with 'HEAD' in the patch function case?